demo/index.html

```diff
@@ -4,12 +4,12 @@
 <head>
   <link href="https://fonts.googleapis.com/css?family=Material+Icons&display=block" rel="stylesheet">
   <script src="https://unpkg.com/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
-  <script type="module" src="../lib/code-sample.js"></script>
+  <script type="module" src="../lib/playground-ide.js"></script>
```

Comment:
I think we should lift the public entrypoints out of the lib/ folder

Reply:
Done.